test/suites/backup/auto-recovery.robot

```diff
@@ -0,0 +1,68 @@
+*** Settings ***
+Documentation       Tests related to auto-recovery functionality
+
+Resource            ../../resources/microshift-host.resource
+Resource            ../../resources/microshift-process.resource
+Library             Collections
+Library             DateTime
+Library             ../../resources/libostree.py
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Test Cases ***
+Backups Are Created Successfully With Expected Schema
+    [Documentation]    Verify that backups intended for auto-recovery are created successfully
+    ...    and they are named with expected schema of DATETIME_DEPLOY-ID for ostree/bootc and DATETIME_MICROSHIFT-VERSION.
+    Stop MicroShift
+    Command Should Work    microshift backup --auto-recovery /var/lib/microshift-auto-recovery
+    ${backup_name}=    Command Should Work    ls /var/lib/microshift-auto-recovery
```

Comment:
I'm not sure what the main purpose of this check is, but seeing file attributes might be helpful in general.
```suggestion
    ${backup_name}=    Command Should Work    ls -l /var/lib/microshift-auto-recovery
```

Reply:
Actually the command is there to get the name of the backup - there's one subdir in the dir and we want its name to decompose into DATE and deploy ID